src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -154,6 +155,30 @@ public AFCalculationResult calculate(final VariantContext vc, final int defaultP
         return new AFCalculationResult(integerAltAlleleCounts, alleles, log10PNoVariant, log10PRefByAllele);
     }
 
+    /**
+     * Calculate the posterior probability that a single biallelic sample is non-ref
```

Comment:
I don't like the phrase `biallelic sample`, but I get that the gist is single-sample.  Maybe biallelic genotype?

Reply:
done